### PR TITLE
Update Airbase to version 127

### PR DESCRIPTION
**Pull request by nineinchnick**
## Description

This version prohibits shaded imports for `org.apache`.

I'll rebase once #12560 gets merged.

> Is this change a fix, improvement, new feature, refactoring, or other?
fix

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
core

> How would you describe this change to a non-technical end user or system administrator?
n/a

## Documentation

(x) No documentation is needed.
( ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(x) No release notes entries required.
( ) Release notes entries required with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: core/trino-main/src/test/java/io/trino/operator/aggregation/listagg/TestListaggAggregationFunction.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ public void testInputEmptyState()
     @Test
     public void testInputOverflowOverflowFillerTooLong()
     {
-        String overflowFillerTooLong = StringUtils.repeat(".", 65_537);
+        String overflowFillerTooLong = repeat(".", 65_537);
```
Comment (electrum) [inline core/trino-main/src/test/java/io/trino/operator/aggregation/listagg/TestListaggAggregationFunction.java:85]: Use the `repeat()` method on `String`
```java
 String overflowFillerTooLong = ".".repeat(65_537);

**Review by electrum — APPROVED** [review]

File: core/trino-main/src/test/java/io/trino/operator/aggregation/listagg/TestListaggAggregationFunction.java (hunk 1 above)
Comment (nineinchnick) [reply core/trino-main/src/test/java/io/trino/operator/aggregation/listagg/TestListaggAggregationFunction.java]: 🤦 thanks!

**Comment by martint** [comment]
@nineinchnick, can you rebase?

**Comment by nineinchnick** [comment]
@martint done!

**Review by hashhar — APPROVED** [review]
Please rebase to make sure CI results are fresh and I'll merge this.

**Pull request merged**